### DISCUSSION TITLE
Check if figures can be improved

### DISCUSSION THREAD

**Discussion by LDSamson** [openpharma/clinsight#211] [Ideas]
Since the `plotly` objects appear to be the largest bottleneck in starting the application, it would be good to explore other options to improve speed and functionality, such as:
- using native `plotly` functions
- using an alternative ([ggiraph](https://www.ardata.fr/ggiraph-book/))

**Comment by jthompson-arcus** [comment]
I have personally found that cutting `ggplot` out helps a lot.

We can also consider not using `plotly` library and use a reduced/partial distribution. This is probably less desirable though.
